### PR TITLE
Add daisy workflows for kokoro signing image build and publish

### PR DESCRIPTION
**Pull request by patelne**
Build a custom VM image kokoro RPM signing.

[go/kokoro-custom-prod-linux-vm](https://goto.google.com/kokoro-custom-prod-linux-vm)

### CODE REVIEW

File: daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+}
+
+# Must have a "kbuilder" user
+useradd -c "kbuilder user" -m -s /bin/bash kbuilder
+groupadd kokoro
+usermod -aG kokoro kbuilder
+
+# kbuilder must allow login with ssh public/private key authentication
+GetMetadataAttribute "kokoro_authorized_keys" && authorized_keys="$attribute_value"
+mkdir ~/.ssh
```
Comment (dntczdx) [inline daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh:21]: it's better to ignore failure when making dir

File: daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+systemctl enable rsyncd.service
+
+# Set network settings
+cat >> /etc/sysconfig/network-scripts/ipcfg-eth0 <<EOF
+IPADDR=169.254.0.2/24
+GATEWAY=169.254.0.1
+DNS=8.8.8.8
+EOF
+
+# Install needed packages
```
Comment (zmarano) [inline daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh:80]: can just make this one line
dnf -y install git nmap-ncat python3-psutil

File: daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+read only = false
+write only = false
+list = false
+uid = kbuilder
+munge symlinks = no
+EOF
+systemctl enable rsyncd.service
+
+# Set network settings
+cat >> /etc/sysconfig/network-scripts/ipcfg-eth0 <<EOF
```
Comment (zmarano) [inline daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh:74]: ipcfg-eth0 -> ifcfg-eth0

File: daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh (hunk 1 above)
Comment (patelne) [reply daisy_workflows/image_build/kokoro/kokoro_image_preparer.sh]: Sorry, didn't see this. Why is that? This directory has to be made for this image to work.

File: daisy_workflows/build-publish/kokoro/kokoro_signing.publish.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
+{
+  "Name": "kokoro-signing",
+  "WorkProject": "gce-image-builder",
+  "ComputeEndpoint": "https://www.googleapis.com/compute/v1/projects/",
+  "DeleteAfter": "24h*30*6",
+  {{if eq .environment "test" -}}
```
Comment (bkatyl) [inline daisy_workflows/build-publish/kokoro/kokoro_signing.publish.json:6]: Will we ever publish this image to more than 1 project? If not this entire if/else approach can be removed.

**Comment by hopkiw** [comment]
Closing in favor of #2000

**Pull request closed without merging**